src/main.c

```diff
@@ -66,6 +67,7 @@ int main() {
 
   // Animation
   while (1) {
+    MikMod_Update();
```

Comment:
This seems to be necessary.
http://web.mit.edu/outland/src/libmikmod-3.1.9/docs/mikmod.html#TOC_67